scripts/src/java/org/oppia/android/scripts/coverage/RunCoverage.kt

```diff
@@ -102,30 +106,30 @@ class RunCoverage(
    * @return a list of lists containing coverage data for each requested test target, if
    *     the file is exempted from having a test file, an empty list is returned
    */
-  fun execute() {
+  fun execute() = runBlocking {
     val testFileExemptionList = loadTestFileExemptionsProto(testFileExemptionTextProto)
       .testFileExemptionList
       .filter { it.testFileNotRequired }
       .map { it.exemptedFilePath }
 
     if (filePath in testFileExemptionList) {
       println("This file is exempted from having a test file; skipping coverage check.")
-      return
-    }
+    } else {
+      val testFilePaths = findTestFile(repoRoot, filePath)
+      if (testFilePaths.isEmpty()) {
+        error("No appropriate test file found for $filePath")
+      }
 
-    val testFilePaths = findTestFile(repoRoot, filePath)
-    if (testFilePaths.isEmpty()) {
-      error("No appropriate test file found for $filePath")
-    }
+      val testTargets = bazelClient.retrieveBazelTargets(testFilePaths)
 
-    val testTargets = bazelClient.retrieveBazelTargets(testFilePaths)
+      val deferredCoverageReports = testTargets.map { testTarget ->
+        runCoverageForTarget(testTarget)
```

Comment:
```suggestion
        runCoverageForTarget(testTarget).await()
```

As we discussed, we should run these in a deterministic sequential order rather than relying on Bazel to synchronize them.

This avoids needing the ``awaitAll()`` below, as well.

Reply:
Got it, removed all async await operations.